diff --git a/node_modules/gatsby-source-drupal/gatsby-node.js b/node_modules/gatsby-source-drupal/gatsby-node.js
index 1f12e22..e3623e4 100644
--- a/node_modules/gatsby-source-drupal/gatsby-node.js
+++ b/node_modules/gatsby-source-drupal/gatsby-node.js
@@ -76,11 +76,14 @@ async function worker([url, options]) {
     }
   }
 
+  globalReporter.verbose(`gatsby-source-drupal is requesting URL`, url)
+
   const response = await got(url, {
     agent,
     cache: false,
     // request: http2wrapper.auto,
     // http2: true,
+    timeout: 60000,
     ...options
   });
   httpSpan.setTag(SemanticAttributes.HTTP_STATUS_CODE, response.statusCode);
diff --git a/node_modules/gatsby-source-drupal/normalize.js b/node_modules/gatsby-source-drupal/normalize.js
index 3c99a24..9a6c553 100644
--- a/node_modules/gatsby-source-drupal/normalize.js
+++ b/node_modules/gatsby-source-drupal/normalize.js
@@ -96,6 +96,9 @@ exports.downloadFile = async ({
 }) => {
   // handle file downloads
   if (isFileNode(node)) {
+
+    console.log(`download file node`, node)
+
     let fileType;
     let fileUrl = node.url;
 
@@ -128,6 +131,8 @@ exports.downloadFile = async ({
       reporter
     });
 
+    console.log('created remote file node', fileNode)
+
     if (fileNode) {
       node.localFile___NODE = fileNode.id;
     }
