diff --git a/node_modules/gatsby-source-filesystem/extend-file-node.js b/node_modules/gatsby-source-filesystem/extend-file-node.js
index dd5550d..febdac5 100644
--- a/node_modules/gatsby-source-filesystem/extend-file-node.js
+++ b/node_modules/gatsby-source-filesystem/extend-file-node.js
@@ -35,7 +35,7 @@ module.exports = ({
           });
         }
 
-        return `${pathPrefix}/static/${fileName}`;
+        return `${pathPrefix}/static/${encodeURI(fileName)}`;
       }
     }
   };
