diff --git a/node_modules/gatsby-source-drupal/gatsby-node.js b/node_modules/gatsby-source-drupal/gatsby-node.js
index f1b2379..943575b 100644
--- a/node_modules/gatsby-source-drupal/gatsby-node.js
+++ b/node_modules/gatsby-source-drupal/gatsby-node.js
@@ -57,7 +57,8 @@ exports.sourceNodes = async ({
     concurrentFileRequests,
     disallowedLinkTypes,
     skipFileDownloads,
-    fastBuilds
+    fastBuilds,
+    requestConfig
   } = pluginOptions;
   const {
     createNode,
@@ -65,6 +66,10 @@ exports.sourceNodes = async ({
     touchNode
   } = actions;
 
+  if (!requestConfig) {
+     requestConfig = {}
+  }
+
   if (webhookBody && Object.keys(webhookBody).length) {
     const changesActivity = reporter.activityTimer(`loading Drupal content changes`, {
       parentSpan
@@ -137,7 +142,8 @@ exports.sourceNodes = async ({
       const data = await axios.get(`${baseUrl}/gatsby-fastbuilds/sync/${lastFetched}`, {
         auth: basicAuth,
         headers,
-        params
+        params,
+        ...requestConfig
       });
 
       if (data.data.status === -1) {
@@ -223,7 +229,8 @@ exports.sourceNodes = async ({
     const data = await axios.get(`${baseUrl}/${apiBase}`, {
       auth: basicAuth,
       headers,
-      params
+      params,
+      ...requestConfig
     });
     allData = await Promise.all(_.map(data.data.links, async (url, type) => {
       if (disallowedLinkTypes.includes(type)) return;
@@ -250,7 +257,8 @@ exports.sourceNodes = async ({
           d = await axios.get(url, {
             auth: basicAuth,
             headers,
-            params
+            params,
+            ...requestConfig
           });
         } catch (error) {
           if (error.response && error.response.status == 405) {
